### DISCUSSION TITLE
The sending of "Webhook" is irregular and never sends all the bounces

### DISCUSSION THREAD

**Discussion by Orminor77** [postalserver/postal#2097] [Help with using Postal] [answered]
Hello,

I am contacting you because I would like to know the solution to correct the problem indicated in the subject of the ticket.

As you can see in the attached screenshot (https://ibb.co/5253yJ7) it is the bounce rate and below the number of bounce messages (5065) that should be sent via the webhook.

Here is the configuration of my webhook (https://ibb.co/tQFdQpV) the bounce should normally be sent in full to Mailwizz, and after several tests the problem is 100% from Postal.

And here the webhook history page, we see that the requests are sent well however only 729 requests were made on 5065 bounces (https://ibb.co/4P0wvZF), as you can see I chose only 4 options, but even by choosing, "Yes in EVENTS" it does not change anything to the bounces in the end.

I'm looking for a solution to solve this problem and to get all the bounces send via webhook, 5065 bounce in this case, or an alternative to get all the bounces in the bounce page via API for example.

Thanks in advance for your help!

**Comment by willpower232** [comment, marked as answer]
Looking at the timestamps in your screenshots, they may have been happening around the same time as you opened this discussion, did they all get received in the end? I presume the worker(s) for your installation might be quite busy and you may be well served by increasing the number if that was the case.
